Assets/Source/Components/UI/Base/HUDBase.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Assets.Source.Components.Actor;
+using Assets.Source.Components.Base;
+using Assets.Source.Components.Player;
+using Assets.Source.Constants;
+using UnityEngine;
+
+namespace Assets.Source.Components.UI.Base
+{
+    /// <summary>
+    /// The base class that all HUD elements should use if they need to update based on player data.
+    /// This class provides consistent access to the player and helper methods.
+    /// </summary>
+    public class HUDBase : ComponentBase
```

Comment:
I don't think this class provides much value.  Saving a few lines of code doesn't warrant a hierarchical relationship imo

Reply:
also it needs to have "Component" or "Behavior" in the name

Reply:
Yeah i guess i thought there'd be more but there really wasn't. Removing.